keras/backend/theano_backend.py

```diff
@@ -782,7 +782,7 @@ def dropout(x, level, seed=None):
     if level < 0. or level >= 1:
         raise Exception('Dropout level must be in interval [0, 1[.')
     if seed is None:
-        seed = np.random.randint(10e6)
+        seed = np.random.randint(1,10e6+1) 
```

Comment:
My apologies for my coding style. I have changed the range for randint to (1, 10e6) and follow the PEP8 style.
I did run into this problem while I was compiling my model and I agree that it is extremely unlikely, and here is the error message I received:
```
Traceback (most recent call last):
  File "cnn_keras_mnist.py", line 59, in <module>
    cnn.compile(loss = "categorical_crossentropy", optimizer = 'adadelta', metrics = ['accuracy'])
  File "/Users/xxx/anaconda/lib/python2.7/site-packages/keras/models.py", line 339, in compile
    **kwargs)
  File "/Users/xxx/anaconda/lib/python2.7/site-packages/keras/engine/training.py", line 510, in compile
    masks = self.compute_mask(self.inputs, mask=None)
  File "/Users/xxx/anaconda/lib/python2.7/site-packages/keras/engine/topology.py", line 1914, in compute_mask
    output_tensors, output_masks, output_shapes = self.run_internal_graph(inputs, masks)
  File "/Users/xxx/anaconda/lib/python2.7/site-packages/keras/engine/topology.py", line 2042, in run_internal_graph
    output_tensors = to_list(layer.call(computed_tensor, computed_mask))
  File "/Users/xxx/anaconda/lib/python2.7/site-packages/keras/layers/core.py", line 87, in call
    x = K.in_train_phase(K.dropout(x, level=self.p), x)
  File "/Users/xxx/anaconda/lib/python2.7/site-packages/keras/backend/theano_backend.py", line 786, in dropout
    rng = RandomStreams(seed=seed)
  File "/Users/xxx/anaconda/lib/python2.7/site-packages/theano/sandbox/rng_mrg.py", line 1134, in __init__
    self.set_rstate(seed)
  File "/Users/xxx/anaconda/lib/python2.7/site-packages/theano/sandbox/rng_mrg.py", line 1145, in set_rstate
    raise ValueError('seed should not be 0', seed)
ValueError: ('seed should not be 0', 0)
```